Core concepts - Minor changes (#1)
deity-io/falcon-docs: docs/getting-started/requirements.md

title: Requirements
---

## Minimal Requirements to run Falcon

- [NodeJS](https://nodejs.org) (>=8.10)
- [Yarn](https://yarnpkg.com) (Optional, but highly recommended)

> To install Yarn, see https://yarnpkg.com/lang/en/docs/install

## Requirements for Falcon local development

- [NodeJS](https://nodejs.org) (>=8.10)
- [Yarn](https://yarnpkg.com) (>=1.12)

> > We run our tests against major Node versions - 8, 10 and 11

Reviewer: ```suggestion
> Falcon Tests are ran against major Node version - 8, 10 and 11.
```
Let's keep documentation in "3rd person"
